projects/robots/robotis/darwin-op/worlds/darwin-op-hinge-vs-hinge2.wbt

```diff
@@ -0,0 +1,38 @@
+#VRML_SIM R2021b utf8
+WorldInfo {
+  info [
+    "ROBOTIS OP2 robot, comparison between the implementation using HingeJoints and Hinge2Joints."
+    "The ROBOTIS OP2 robot simulation model can be programmed using the ROBOTIS OP2 motions files and the ROBOTIS OP2 walk control library."
+  ]
+  title "ROBOTIS OP2 modeling comparison"
+  basicTimeStep 8
```

Comment:
I would leave a time step of 16 ms if it works here.
```suggestion
  basicTimeStep 16
```

Reply:
Since it doesn't have backlash enabled by default it can be 16 yes.